### PR TITLE
48098_Update_Profile with modification of Sign out

### PR DESCRIPTION
**Pull request by AVI5HEK**

### CODE REVIEW

File: app/src/main/java/com/framgia/elsytem/LoginActivity.java [hunk 1]
```diff
@@ -46,12 +47,15 @@ protected void onCreate(Bundle savedInstanceState) {
         toolbar.setLogo(R.mipmap.ic_launcher);
         session = new SessionManager(getApplicationContext());
         // check in session if user is logged in. If so, go to profile activity
-        if (session.isLoggedIn()) {
+        if (session.isLoggedIn() && ((session.getUserDetails()).get(KEY_REMEMBER_ME)).equals("1")) {
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/LoginActivity.java]: you should method like this in SessionManger, eg:
public boolean isLoggedInAndRemember(){
return session.isLoggedIn() && ((session.getUserDetails()).get(KEY_REMEMBER_ME)).equals("1"); 
}

File: app/src/main/java/com/framgia/elsytem/LoginActivity.java [hunk 2]
```diff
@@ -46,12 +47,15 @@ protected void onCreate(Bundle savedInstanceState) {
...
             //go to ProfileActivity
             // going to UpdateProfileActivity for testing purpose
             Intent i = new Intent(getApplicationContext(), UpdateProfileActivity.class);
             startActivity(i);
             finish();
+        } else if (session.isLoggedIn() && ((session.getUserDetails()).get("rememberMe")).equals
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/LoginActivity.java]: same as above

File: app/src/main/java/com/framgia/elsytem/RegisterActivity.java [hunk 3]
```diff
@@ -83,10 +81,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                 int columnIndex = cursor.getColumnIndex(filePathColumn[0]);
                 imgDecodableString = cursor.getString(columnIndex);
                 cursor.close();
-                ImageView imgView = (ImageView) findViewById(R.id.image_upload_avatar);
+                /*ImageView imgView = (ImageView) findViewById(R.id.image_upload_avatar);
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/RegisterActivity.java]: remove code comment please

File: app/src/main/java/com/framgia/elsytem/RegisterActivity.java [hunk 4]
```diff
@@ -45,13 +43,13 @@ protected void onCreate(Bundle savedInstanceState) {
         toolbar.setLogo(R.mipmap.ic_launcher);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         getSupportActionBar().setDisplayShowHomeEnabled(true);
-        ImageView avatar = (ImageView) findViewById(R.id.image_upload_avatar);
+        /*ImageView avatar = (ImageView) findViewById(R.id.image_upload_avatar);
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/RegisterActivity.java]: remove this please

File: app/src/main/java/com/framgia/elsytem/UpdateProfileActivity.java [hunk 5]
```diff
@@ -38,9 +54,20 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayShowHomeEnabled(true);
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/UpdateProfileActivity.java]: please beark onCreate to small method by function, e.g: updateActionBar(), findViews(),...

File: app/src/main/res/layout/content_register.xml [hunk 6]
```diff
@@ -99,7 +99,7 @@
         android:layout_below="@id/image_full_name"
         android:hint="@string/edit_text_full_name_activity_register"
         android:inputType="textPersonName"/>
-    <ImageView
+    <!--<ImageView
```
Comment (nvmanh) [inline app/src/main/res/layout/content_register.xml]: remove this

File: app/src/main/java/com/framgia/elsytem/LoginActivity.java [hunk 7]
```diff
@@ -147,16 +148,18 @@ protected void onPostExecute(String result) {
...
-                        String name = new JSONObject(new JSONObject(result).getString("user")).getString
-                                ("name");
-                        String email = new JSONObject(new JSONObject(result).getString("user")).getString
-                                ("email");
-                        if (!name.isEmpty() && !email.isEmpty())
-                            session.createLoginSession(name, email);
-                        Log.e(TAG + " Name: ", name);
-                        Log.e(TAG + " Email: ", email);
-                    }
+                    String name = new JSONObject(new JSONObject(result).getString("user")).getString
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/LoginActivity.java]: please use Gson for parsing json data to short code

**Comment by AVI5HEK** [comment]
Changed codes in ProfileActivity, UpdateProfileActivity, SessionManager, improved GUI on content_profile.xml and so on just to make the app a bit more functional. Sorry for all that. :(

**Comment by nvmanh** [comment]
OK!

**Pull request merged**